### ISSUE TITLE
CW mode very choppy 

### ISSUE THREAD

**Issue by clytle374** [AlexandreRouma/SDRPlusPlus#572]
Running the latest git source, be this way for a week or more.  When selecting CW mode the display and audio drop to about one frame per second.  No issues on other modes.  Have to work the next few day, might be a while before I can respond to questions.  

Also I have channels ghosting in sdr++ that I don't have in gqrx.  Any suggestions?
Thanks, Cory

**Comment by AlexandreRouma** [comment]
your computer can't handle the bandwidth you've selected for the CW demodulatot

**Comment by clytle374** [comment]
Thanks, there are 3 bandwidths.  Sadly I don't understand what is what there. 
The first is directly under the limesdr device.  Lowering way down helps it.
Another bandwidth a few more lines down.  If I don't keep this way up, I lose most of my signals. This doesn't affect hangs.
The one below CW selection maxes out at 500, I have to keep it up around 500 even with the top setting turned down.  This never happened until recently.
 Machine is a 6 core  Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
I've got the limesdr amplified and with a LPF now, building a new loop with more amplification.
Thanks

**Comment by AlexandreRouma** [comment]
what sample rate is the LimeSDR running at?

**Comment by AlexandreRouma** [comment]
Also for the "channel ghosting" thing, send a screenshot of both GQRX and SDR++
My guess is you either messed with the bandwidth setting in the source menu or have the gain too high

**Comment by clytle374** [comment]
Sorry, I got sidetracked with work and other issues.  I've messed with all the settings. All 3 gains are maxed on gqrx as the limesdr receive is so weak at these frequencies.  My low pass filter has been corrected and tested, FM no longer getting into the receiver.   I don't think it's a AM broadcast station as I can't receive any currently.  

![gqrxwwv](https://user-images.githubusercontent.com/18334763/149618206-bbcd9615-5873-4e4b-9736-c0bf3ef4e80d.png)
Here is WWV that I can't pick up with sdr++
![sdr++](https://user-images.githubusercontent.com/18334763/149618210-3cdbd0eb-52cb-4348-803f-f01ee5df6073.png)
Here is the screenshot with CW working, I have turned the setting down right below the limesdr device in the menu.  Before I left it a max all the time, now it works fine at max in any mode other than CW.
Building a new amplified antenna, probably going to quit fighting this until I get enough signal into the lime to work better.  
Thanks

**Comment by clytle374** [comment]
Apologies for the previous post, I didn't realize until just now I didn't post the 2 pictures of the same frequencies. 

The first number is the sample rate of the Limesdr?  I brought it down to 5MS/s, at the same time I set bandwidth to auto and I got absolutely nothing. Again, I don't completely the bandwidth setting.  But setting it to low numbers renders receiver completely deaf, it's only mostly deaf in HF stock.  Samples per second from the SDR makes sense. I finally took some time to investigate more as the loop antenna amplifier project is making me crazy, lol.  

Here is what I figured out as I copied settings from gqrx and still had nothing on SDR++.  

gqrx
![gqrx](https://user-images.githubusercontent.com/18334763/149893825-aa55a7ef-800f-4828-9016-7939dad36a07.png)


sdr++ on auto
![sdr++auto](https://user-images.githubusercontent.com/18334763/149894646-c980beb0-da1b-4663-ad04-11d4aba64d92.png)
 

sdr++ after manually changing bandwidth back to maximum
![sdr++manual](https://user-images.githubusercontent.com/18334763/149893992-485ffffc-0b18-4692-afce-7ca8dc9d3b33.png)

Advice or reading materials?  Am I doing something stupid? 
Thanks, Cory
